README.md

```diff
@@ -8,3 +8,165 @@ You can choose to use either the:
 - [React COGS SDK](./packages/javascript/README.md)
 
 Learn more in the [COGS technical documentation](https://docs.cogs.show).
+
+## Quick start
```

Comment:
While the quick start is sort of useful, if developers only see this and not the "Getting Started" section below then they won't be set up correctly as it's missing important steps if you just install the library and try to use it.

Personally, I would remove this section completely. You then start with explaining that the library is used for (creating either custom content or a plugin) and then follow it up with how to get started.

Reply:
Hmm yeah that's fair. It probably makes sense to look at the relevant SDK readme page to see how to add it.